nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/ProcessGroupResource.java

```diff
@@ -314,7 +430,7 @@ public Response updateProcessGroup(
                     Authorizable authorizable = lookup.getProcessGroup(id).getAuthorizable();
                     authorizable.authorize(authorizer, RequestAction.WRITE, NiFiUserUtils.getNiFiUser());
                 },
-                null,
+                () -> serviceFacade.verifyUpdateProcessGroup(requestProcessGroupDTO),
```

Comment:
Good call. Originally, I had it updating the variables here but then refactored quite a bit. I think it's best to go ahead and leave in the addition of the verifyUpdateProcessGroup method, but make the verification a NOP. We do this in a few other places, as well, so I feel it's best to leave the 'plumbing' there, since it's already been built. It will make it easier to update later.